src/functions-and-arrays.js

```diff
@@ -29,8 +91,37 @@ const wordsUnique = [
   'bring'
 ];
 
+
+var uniquifyArray = function(array){
+  //var storedLength = array.length;
+  if (array.length === 0) {
+    return null;
+  }
+  for (i = 0; i < array.length; i++) {
+    for (j = i+1; j < array.length; j++) {
+       if( array[i]===array[j] ){
+          array.splice(j,1);
+       }
+    }
+  }
+  return array;
+}
+
 // Iteration #6: Find elements
 const wordsFind = ['machine', 'subset', 'trouble', 'starting', 'matter', 'eating', 'truth', 'disobedience'];
+var doesWordExist = (word, array) =>{
```

Comment:
Jasmine throws two errors and this is only due to the order of the parameters you're passing.

Just put array first and word second and it will work :)